client/core/core.go

```diff
@@ -3221,8 +3223,7 @@ func (c *Core) initialize() {
 			continue
 		}
 		// Wallet is loaded from the DB, but not yet connected.
-		c.log.Infof("Loaded %s wallet configuration. Deposit address = %s",
-			unbip(aid), dbWallet.Address)
+		c.log.Infof("Loaded %s wallet configuration.", unbip(aid))
```

Comment:
The deposit address isn't validated until connect, so we don't want to show it here.